tests/ruby/RuboCopBearTest.py

```diff
@@ -1,19 +1,58 @@
+import os
+
 from bears.ruby.RuboCopBear import RuboCopBear
 from tests.LocalBearTestHelper import verify_local_bear
 
-good_file = """def bad_name
+good_file = """def good_name
```

Comment:
ahh i guess this change belonged in a different commit? but i had to change this test anyway, so i figured i could keep it here?